apps/api/src/resources/User/resolver.ts

```diff
@@ -104,6 +106,43 @@ export class UserResolver extends ResourceResolver<resourceType>(
     return user.save();
   }
 
+  @Authorized("update:user_expiration_date")
+  @Mutation((_: void) => User)
+  public async updateExpirationDate(
+    @Arg("userId") userId: string,
+    @Arg("newExpirationDate") newExpirationDate: Date,
+  ): Promise<User> {
+    const user: User = await User.findOneOrFail({ id: userId });
+    user.membershipExpiration = newExpirationDate;
+    return user.save();
+  }
+
+  @Authorized("update:user_shirt_received")
+  @Mutation((_: void) => User)
+  public async updateShirtReceived(
+    @Arg("userId") userId: string,
+    @Arg("updatedShirtStatus") updatedShirtStatus: boolean,
+  ): Promise<User> {
+    const user: User = await User.findOneOrFail({ id: userId });
+    user.shirtReceived = updatedShirtStatus;
+    return user.save();
+  }
+
+  @Authorized("reset:user_shirt_received")
+  @Mutation((_: void) => [User], { nullable: true })
```

Comment:
leave a comment here denoting why it can be nullable

Reply:
I no longer have any idea why-- but I can test this unless I re-generate, right?

Reply:
Did removing this nullable field still result in functional code